multidim_threshold/threshold.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+from copy as deepcopy
+
+from .refine import box_edges
```

Comment:
Just as a preference, I prefer not to use relative imports.
```python
from multidim_threshold.refine import box_edges
```

Reason has to do with historical Python 2 functionality as well as some general style guidelines.